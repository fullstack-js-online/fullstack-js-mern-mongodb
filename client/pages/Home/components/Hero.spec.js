import React from 'react'
import { mount } from 'enzyme'

import Hero from './Hero'

describe('The Hero component', () => {
  it('should match snapshot', () => {
    expect(mount(<Hero />)).toMatchSnapshot()
  })
})
