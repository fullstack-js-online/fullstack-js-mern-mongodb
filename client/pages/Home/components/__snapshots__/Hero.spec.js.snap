// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The Hero component should match snapshot 1`] = `
<Hero>
  <div
    className="w-full h-64 flex items-center justify-center"
  >
    <h1
      className="text-5xl font-bold"
    >
      FULLSTACK-JS MERN STARTER
    </h1>
  </div>
</Hero>
`;
